{
  "path": "/documentation/foundation/urlerror/backgroundtaskcancelledreason/userforcequitapplication",
  "type": "Enumeration Case",
  "name": "URLError.BackgroundTaskCancelledReason.userForceQuitApplication",
  "desc": "A reason that indicates the system canceled the background task because the user force-quit the application.",
  "items": [],
  "declaration": "case userForceQuitApplication"
}